misc/librenms-irc.service

```diff
@@ -0,0 +1,12 @@
+# /etc/systemd/system/librenms-irc.service 
+
+librenms-irc.service
+[Unit]
+Description=Librenms IRC bot
+
+[Service]
+ExecStart=/opt/librenms/irc.php
```

Comment:
`/etc/sysconfig/librenms` was just a guess.  Is there a more distro neutral location?